src/miral/symbols.map

```diff
@@ -571,8 +571,11 @@ MIRAL_5.5 {
 global:
   extern "C++" {
     "miral::CursorScale::CursorScale(miral::live_config::Store&)";
+    "miral::CursorScale::CursorScale(miral::live_config::Store&)";
+    "miral::InputConfiguration::InputConfiguration(miral::live_config::Store&)";
     "miral::InputConfiguration::InputConfiguration(miral::live_config::Store&)";
     "miral::OutputFilter::OutputFilter(miral::live_config::Store&)";
+    "miral::OutputFilter::OutputFilter(miral::live_config::Store&)";
```

Comment:
Why are we duplicating these entries? Or rather, we have duplicates that are now sorted together. But let's dedup as we're changing this